settings/ci.mdx

```diff
@@ -0,0 +1,114 @@
+---
+title: CI Checks
+description: "Use Mintlify's CI tools to lint your docs"
+icon: 'circle-check'
+---
+
+<Warning>
+  This feature is only available for customers using GitHub. If you'd like it enabled for other platforms, please provide [feedback](https://feedback.mintlify.com/roadmap).
+</Warning>
```

Comment:
could we also add a note about it being available on paid plans?

Reply:
Nice catch! Added